transforms/setters/api.graphql

```diff
@@ -0,0 +1,41 @@
+# This example demonstartes mapping a JSON response to the fields of the GraphQL.
+
+type Customer {
```

Comment:
Actually, can we have the GraphQL object have the natural names and the response have "weird" names